.github/workflows/build.yml

```diff
@@ -85,3 +85,17 @@ jobs:
           ${{ matrix.host }}
           ./zcutil/build.sh
           -j"$(nproc 2> /dev/null || sysctl -n hw.ncpu)"
+
+      - name: Build zcashd with wallet disabled
+        run: >
+          CONFIGURE_FLAGS="--disable-wallet"
+          ${{ matrix.host }}
+          ./zcutil/build.sh
+          -j"$(nproc 2> /dev/null || sysctl -n hw.ncpu)"
+
+      - name: Build zcashd with mining disabled
+        run: >
+          CONFIGURE_FLAGS="--disable-mining"
+          ${{ matrix.host }}
+          ./zcutil/build.sh
+          -j"$(nproc 2> /dev/null || sysctl -n hw.ncpu)"
```

Comment:
```suggestion
      - name: Build zcashd with mining disabled
        run: >
          CONFIGURE_FLAGS="--disable-mining"
          ${{ matrix.host }}
          ./zcutil/build.sh
          -j"$(nproc 2> /dev/null || sysctl -n hw.ncpu)"

      - name: Build zcashd with both wallet and mining disabled
        run: >
          CONFIGURE_FLAGS="--disable-wallet --disable-mining"
          ${{ matrix.host }}
          ./zcutil/build.sh
          -j"$(nproc 2> /dev/null || sysctl -n hw.ncpu)"
```

Reply:
While there are edge cases where this might help, I don't think testing both at once is going to give us much more useful information than testing them individually. And it seems that at least adding `--disable-wallet` forces a full recompile of `zcashd` in that it doubles the build time (though maybe this will decrease the next time the cache gets used).

If anything, I'd prefer to spend this CI time running a `--with-libs` build, to confirm that `libzcash_script` builds work.